### ISSUE TITLE
Articles do not contain "other uses" section at the top

### ISSUE THREAD

**Issue by ashr314** [wikispeedruns/wikipedia-speedruns#506]
Original Wikipedia article contains links to Pilgrim and Pilgrim (disambiguation) at the top, while the speedruns page does not. It's worth noting that in this particular case the link to Pilgrim is only in the omitted section and therefore unreachable in this path during a speedrun.
![image](https://github.com/wikispeedruns/wikipedia-speedruns/assets/40901702/93c6fcc1-1c13-42b7-8c5d-616a22926ada)
![image](https://github.com/wikispeedruns/wikipedia-speedruns/assets/40901702/92ca761b-2c72-4255-9839-6eef6f27458d)

**Comment by bricehalder** [comment]
Hmm this might be intentional? cc @mliu59

**Comment by mliu59** [comment]
@ashr314 This section is currently omitted intentionally. When we first created the game, we removed the subheading/disambiguation section because we thought that it made the game too easy. However, I can see how it could bring needless frustration to players. Plus, I don't think we're doing a thorough job of blocking disambiguation pages anyway, since they are usually accessible via normal links. 

@bricehalder @dqian3 thoughts?
